packages/core/src/components/ui/Newsletter/Newsletter.tsx

```diff
@@ -124,10 +92,10 @@ const Newsletter = forwardRef<HTMLFormElement, NewsletterProps>(
     const { subscribeUser, loading, data } = useNewsletter()
     const nameInputRef = useRef<HTMLInputElement>(null)
     const emailInputRef = useRef<HTMLInputElement>(null)
-    const subscriptionButtonLabel = useMemo(
-      () => (loading ? subscribeButtonLoadingLabel : subscribeButtonLabel),
-      [loading, subscribeButtonLabel, subscribeButtonLoadingLabel]
-    )
+    // const subscriptionButtonLabel = useMemo(
+    //   () => (loading ? subscribeButtonLoadingLabel : subscribeButtonLabel),
+    //   [loading, subscribeButtonLabel, subscribeButtonLoadingLabel]
+    // )
```

Comment:
This was unused code. I'm assuming we've kept it here for some reason, so I didn't delete it